portstyle.css

```diff
@@ -91,112 +94,105 @@ body {
 	align-self: center;
 }
 
+
 .textAboutMe {
 	display: flex;
 	align-items: center;
 }
 
-
-.threeBoxLayout {
-	display: grid;
-	grid-template-columns: 33.3% 33.3% 33.3%;
-	padding: 20px 20%;
-	
-}
-
-
 .portfolioTitle {
 	display: flex;
 	justify-content: center;
-	margin-top: 50px;
+	padding-top: 50px;
+	background: #658fa7;
 }
 
 
-
-.leftBox {
-	display: flex;
-	padding: 20px 20px;
-	flex-direction: column;
-	justify-content: center;
+.threeBoxLayout {
+	display: grid;
+	grid-template-columns: 33.3% 33.3% 33.3%;
+	padding: 10px 20% 100px 20%;
+	background: #658fa7;
 }
 
-.middleBox {
-	display: flex;
-	padding: 20px 20px;
-	flex-direction: column;
-	justify-content: center;
-}
 
-.rightBox {
+.threeBoxLayout .portfolioBox {
 	display: flex;
 	padding: 20px 20px;
 	flex-direction: column;
 	justify-content: center;
 }
 
-
-.leftBelowBox {
-	display: flex;
-	padding: 20px 20px;
-	flex-direction: column;
-	justify-content: center;
- 
+.background {
+	display: grid;
+	grid-template-rows: 1fr 1fr;
+	background: #78a6b8;
+	align-items: flex-start;
```

Comment:
You've got align-items which is a flex-box property on a display grid?

Reply:
Another bit of legacy code that I did not need, thank you.